mediapipe/examples/desktop/face_mesh_dll/face_mesh_lib.cpp

```diff
@@ -106,6 +135,14 @@ MPFaceMeshDetector::DetectFacesWithStatus(const cv::Mat &camera_frame,
   MP_RETURN_IF_ERROR(graph.AddPacketToInputStream(
       kInputStream, mediapipe::Adopt(input_frame.release())
                         .At(mediapipe::Timestamp(frame_timestamp_us))));
+  // Send calibration packet into the graph.
+  //MP_RETURN_IF_ERROR(graph.AddPacketToInputStream(
+  //    kInputStream_camera_matrix, mediapipe::MakePacket<cv::Mat>(m_cameraMatrix)
+  //    .At(mediapipe::Timestamp(frame_timestamp_us))));
+  // Send fps packet into the graph.
+  MP_RETURN_IF_ERROR(graph.AddPacketToInputStream(
+      kInputStream_fps, mediapipe::MakePacket<int>(fps)
```

Comment:
I think this should be done in the constructor rather than here. Filter doesn't need to adapt to such small changes of fps.

Reply:
Yeah, but we don't know right fps at start